app/assets/stylesheets/jobs.css

```diff
@@ -0,0 +1,24 @@
+.jobs {
+  float: right;
+  width: 80%;
+}
+
+table {
+  width: 100%;
+}
+
+.index-job {
+  width: 100%;
+  border: solid;
+  border-color: black;
+  border-width: 1px;
+}
+
+.index-job:hover {
+  background-color: #BDC3C7;
```

Comment:
Maybe set up variables for colors at the top so it makes it easier to see immediately what color everything is? As discussed here: https://css-tricks.com/set-up-variables-for-colors-in-css/